src/components/mindset/NextActions.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from "react"
+import { makeStyles } from '@material-ui/core/styles';
+
+// Material UI
+import Button from "@material-ui/core/Button"
+
+const useStyles = makeStyles(theme => ({
+  button: {
+    margin: theme.spacing(1),
+  },
+}));
+
+const ResultsNote = () => {
+  const classes = useStyles();
+  return (
+    <Fragment>
+      <Button variant="outlined" href="/mindsets" key='see-all' className={classes.button}>
+        See All Mindsets
+      </Button>
+      <Button variant="outlined" href="/" key='start-over' className={classes.button}>
+        Start Over
+      </Button>
+    </Fragment>
```

Comment:
I'm not sure this is the right `href` for start over.

Reply:
You'll want to actually handle this a bit differently I think. MaterialUI will lose the redux store if `href` is used, so you'll want to instead pull in the `Link` component from `react-router`, add the property `component={Link}` and `to="/"`.

Also you probably just wanna shoot them over to `/quiz`, it will start them at the first question to go through the quiz again, since they already accepted/declined cookies and accepted the terms, I think we can skip that first part.